src/index.html

```diff
@@ -10,23 +10,49 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Mate academy | airbnb search bar</title>
     <link
       rel="stylesheet"
       href="style.css"
     />
-  </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
+    <link
+      rel="stylesheet"
+      href="search-form/search-form.css"
     />
+  </head>
+  <body class="page">
+    <form
+      class="form page__form form_big"
+      action="GET"
+      data-qa="big"
+    >
+      <label
+        class="form__item1"
+        data-qa="big-bar"
+      >
+        <input
+          class="form__item2 form__item2_big"
```

Comment:
There are many ways to name classes in BEM but for me the best one is when we use double hyphens for modifiers. `form__item2--big`